filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -306,6 +306,8 @@ DECL_DRIVER_API_SYNCHRONOUS_N(void, cancelExternalImage, void*, image)
 DECL_DRIVER_API_SYNCHRONOUS_N(bool, getTimerQueryValue, backend::TimerQueryHandle, query, uint64_t*, elapsedTime)
 DECL_DRIVER_API_SYNCHRONOUS_N(backend::SyncStatus, getSyncStatus, backend::SyncHandle, sh)
 DECL_DRIVER_API_SYNCHRONOUS_N(bool, isWorkaroundNeeded, backend::Workaround, workaround)
+DECL_DRIVER_API_SYNCHRONOUS_N(void, setupExternalResource, intptr_t, externalResource)
+DECL_DRIVER_API_SYNCHRONOUS_N(void, cancelExternalResource, intptr_t, externalResource)
```

Comment:
I suggest getting rid of cancelExternalResource(). We retain when import/setExternalBuffer/setExternalIndexBuffer is called on the public API and release when the driver thread gets to process the relevant command. There is no need for any explicit release logic.

Reply:
[Addressed here](https://github.com/shapr3d/filament/pull/46/commits/913a0b8e121102664cc383ad7041a412d8641d1a).